classes/Parameters/UpgradeFileNames.php

```diff
@@ -163,7 +163,8 @@ class UpgradeFileNames
      * @var array<string, string>
      */
     public static $backup_tmp_files = [
-        'STATE_FILENAME' => self::STATE_FILENAME,
+        // TODO: Needs the split of state files to avoid dropping data useful for the update on the web UI
+        // 'STATE_FILENAME' => self::STATE_FILENAME,
         'FILES_TO_BACKUP_LIST' => self::FILES_TO_BACKUP_LIST,
         'DB_TABLES_TO_BACKUP_LIST' => self::DB_TABLES_TO_BACKUP_LIST,
     ];
```

Comment:
Have you checked if the CLI backup still works?

Reply:
Given the fact the state is not deleted anymore at the end, its behavior is altered on the CLI version as on the web.
It must be changed for the v7 but to avoid massive PRs, that's why the TODO's here.

Reply:
we clean the state before each backup creation in the CLI, for clean previous backup creation. If we remove the state file from the list it will no longer be clean and 2 backup creations in a row will not work properly

Reply:
Yes I saw the comment ^^I just wonder why passing a PR or the CLI does not work well, instead of doing it on another PR directly. this avoids us finding ourselves in an unstable state or a feature is broken